install_files/ansible-base/roles/build-securedrop-app-code-deb-pkg/files/usr.sbin.apache2

```diff
@@ -105,6 +105,7 @@
   /var/lib/securedrop/db.sqlite-journal w,
   /var/lib/securedrop/keys/* rwl,
   /var/lib/securedrop/keys/*.app-staging.* w,
+  /var/lib/securedrop/keys/gpg-agent.conf rw,
```

Comment:
Does `apache` need to be able to write to this file? Because `postinst` is run as `root`, so with `r` might not be needed.